contracts/lib/IdentityLib.sol

```diff
@@ -72,25 +72,23 @@ library IdentityLib {
      * @param _stateContractAddr - address of the State contract
      * @param _identityAddr - address of the Identity contract, which calls this function
      * @param depth - depth of identity SMTs
+     * @param idType - idType
      */
     function initialize(
         Data storage self,
         address _stateContractAddr,
         address _identityAddr,
-        uint256 depth
+        uint256 depth,
+        bytes2 idType
     ) external {
         require(depth <= IDENTITY_MAX_SMT_DEPTH, "SMT depth is greater than max allowed depth");
         self.stateContract = IState(_stateContractAddr);
+        require(self.stateContract.isIdTypeSupported(idType), "id type doesn't exist");
```

Comment:
Should be `id type is not supported`, because State contract doesn't know if it exists on other blockchains.

Reply:
fixed